src/iop/toneequal.c

```diff
@@ -2278,6 +2284,14 @@ void gui_focus(struct dt_iop_module_t *self, gboolean in)
     g->mask_display = FALSE;
     dt_bauhaus_widget_set_quad_active(GTK_WIDGET(g->show_luminance_mask), FALSE);
     dt_dev_reprocess_center(self->dev);
+    dt_control_hinter_message(darktable.control, "");
+  }
+  else
+  {
+    dt_control_hinter_message(darktable.control,
+                              "scroll over image to change tone exposure by 0.25 EV\n"
+                              "shift+scroll to change by 1 EV, and ctrl+scroll to change by 0.1 EV\n"
+                              "A+scroll to zoom");
```

Comment:
similarly here

Reply:
Yikes. What should the text be? Should I try to read the preferences?

Reply:
TBH I'd just remove this line. The same shortcut is also used to pan/zoom while editing masks and there probably isn't enough space to add it to the mask hinter messages as well. I think for some languages even the current mask hinter message doesn't completely fit in the available space.

Reply:
If you really wanted to look it up I think it's stored in ``darktable.control->accels.darkroom_skip_mouse_events``

Reply:
Alright, I just removed the last line.